lib/3scale/backend/configuration.rb

```diff
@@ -55,6 +55,18 @@ def parse_int(value, default)
       config.add_section(:oauth, :max_token_size)
       config.add_section(:master, :metrics)
 
+      config.add_section(
+          :async_worker,
+
+          # Max number of jobs in the reactor
+          :max_concurrent_jobs,
+          # Max number of jobs in memory pending to be added to the reactor
+          :max_pending_jobs,
+          # Seconds to wait before fetching more jobs when the number of jobs
+          # in memory has reached max_pending_jobs.
+          :seconds_before_fetching_more
```

Comment:
Not sure if it's worth it to add an extra dependency for that.

Reply:
Sorry I meant the rename it to throttling

Reply:
:+1: 
I misunderstood you :sweat_smile:

Reply:
I thought about this, but in the end I decided to keep the original name because I think it's more explicit. Throttling is more generic, depending on the algorithm used it can refer to number of requests per unit of time, that plus a burst rate, etc.